.githooks/pre-commit

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+early_exit=0
+
+# Only run if there are files staged for commit
+changed_files=$(git diff --cached --name-only --diff-filter=ACM | paste -s -d' ' -)
+if [ -n "$changed_files" ]; then
+
+  # Run rubocop
+  echo "Running rubocop $changed_files (this may take a few seconds)"
+  if rubocop_results=$(rubocop $changed_files 2>&1); then
+    if $(echo $rubocop_results | grep -q "Offenses:" 2>&1); then
+      >&2 echo "\n** Rubocop found some code style it disagreed with, try autofixing by running **\n  rubocop --auto-correct $changed_files\n"
```

Comment:
I've never seen a redirect `>&2` thing at the beginning, didn't know that was a thing. TIL

Reply:
Yeeaaaahh still not sure how I feel about it.  Trying new things!